### PR TITLE
5955: update Tablesaw_display_as_TableDisplay.ipynb to load from mvn 

### PR DESCRIPTION
**Pull request by jaroslawmalekcodete**

### CODE REVIEW

**Comment by scottdraves** [comment]
FYI @benmccann

File: doc/contents/Tablesaw_display_as_TableDisplay.ipynb [hunk 1]
```diff
@@ -8,49 +8,26 @@
...
-     "output_type": "stream",
-     "text": [
-      "Added jar: [tablesaw-0.7.6.4_Demo.jar]\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "%classpath add jar demoResources/tablesaw-0.7.6.4_Demo.jar\n",
+    "%classpath add jar demoResources/tablesaw_0.10.0_Demo.jar\n",
```
Comment (benmccann) [inline doc/contents/Tablesaw_display_as_TableDisplay.ipynb:30]: what's in this jar? can we add a pointer to the source code?

**Comment by benmccann** [comment]
This is great! Thanks!

File: doc/contents/Tablesaw_display_as_TableDisplay.ipynb (hunk 1 above)
Comment (scottdraves) [reply doc/contents/Tablesaw_display_as_TableDisplay.ipynb:30]: The src for the jar is in the markdown of the notebook, it's more clearly visible here: https://github.com/twosigma/beakerx/blob/73ac34418eeed06df280b774dc51d44857dd5730/doc/contents/Tablesaw_display_as_TableDisplay.ipynb

**Pull request merged**